static/src/styles/core/fonts.styl

```diff
@@ -0,0 +1,3 @@
+// This file must be imported before any files that contain CSS output
+// This is because @import declarations must be the first declarations in CSS files
```

Comment:
With this in mind, is there a better way we should be importing fonts or is this fine?